core-bundle/src/DependencyInjection/Compiler/AddAssetsPackagesPass.php

```diff
@@ -136,4 +136,17 @@ private function getBundlePackageName(string $className): string
 
         return Container::underscore($className);
     }
+
+    private function findBundlePath(array $meta, string $name): ?string
+    {
+        if (is_dir($path = $meta[$name]['path'].'/Resources/public')) {
+            return $path;
+        }
+
+        if (is_dir($path = $meta[$name]['path'].'/public')) {
```

Comment:
We should check this first so the new path wins if there are both the new and the old path, don't you think?

Reply:
no, I don't think so. Because it keeps BC, and because it's the same Symfony does: https://github.com/symfony/symfony/blob/master/src/Symfony/Bundle/FrameworkBundle/Command/AssetsInstallCommand.php#L136

Reply:
That is an if condition, which does not affect the behavior of the application. So far, we always followed the logic "if the new resource exists, use it". See:

https://github.com/contao/contao/blob/ab3da983e6ef5a19427f2456628ef45e18d9ffe3/manager-bundle/src/HttpKernel/ContaoKernel.php#L342-L351

We never use the old resource and ignore the new one if it exist!

Reply:
Please read the Symfony code again. The if-condition is assigning the path, and only the first if-condition is executed if it matches.

I know the root project behaves differently. But this is for bundles, and the current way is not deprecated. It's also still the way that is documented in the Symfony docs.

Reply:
You are right. My bad. 🙈